pages/common/[.md

```diff
@@ -12,13 +12,9 @@
 
 `[ "${{variable}}" -{{eq|ne|gt|lt|ge|le}} {{integer}} ]`
 
-- Test if the specified variable has a [n]on-empty value:
+- Test if the specified variable has an empty[z]/[n]on-empty value:
```

Comment:
`empty` doesn't contain `z`, so the bracket notation doesn't make a lot of sense. Also, if there are not more than 8 examples, why should we condense them?